_includes/carpentries-commons.html

```diff
@@ -0,0 +1,34 @@
+{% if site.carpentry == "swc" or site.carpentry == "dc" or
+   site.carpentry == "lc" or site.carpentry == "cp" %}
+
+{% comment %}
+We have no notice for official lessons
+{% endcomment %}
+
+{% elsif site.carpentry == "incubator" %}
+
+
+<div class="alert alert-info text-center" role="alert">
+  This lesson is part of
+  <a href="https://github.com/carpentries-incubator/proposals/#the-carpentries-incubator">
+    The Carpentries Incubator</a>, a place to share and use each other's
+  Carpentries-style lessons. <strong>This lesson has not been reviewed by and is
+  not endorsed by The Carpentries</strong>.
+</div>
```

Comment:
@ErinBecker please confirm that this wording is OK

Reply:
Thanks @fmichonneau - this is good.